less/styles.less

```diff
@@ -294,4 +294,15 @@ div.card-panel.panel div.panel-body.panel-body-with-img {
   color: #F98200;
   font-weight: 700;
   margin-top: 10px;
+}
+
+// Hotfixes
```

Comment:
I feel this is a hack. I think we simply need to remove the btn-sm class on 

<button type="button" class="btn btn-sm btn-warning dropdown-toggle details-link" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            <span class="caret"></span>
            <span class="sr-only">Toggle dropdown</span>
          </button>

Reply:
We can't modify the class list because the shortcode is defined on EFHST. What you brought up, I did in a [patch on EFHST](https://gitlab.eclipse.org/eclipsefdn/it/webdev/hugo-solstice-theme/-/commit/bcdbbb45d95bf3081b80cecdd5a72b1855d732cf) but that would require an upgrade to the Hugo version. I was thinking this could be a temp fix.

Reply:
I am fine with a temp solution as long as you create an issue to clean up once we upgrade the hugo theme!

Reply:
Here is the issue to keep track of the cleanup: https://github.com/jakartaee/jakarta.ee/issues/1649